### ISSUE TITLE
FACTS Use Tutorial

### ISSUE THREAD

**Issue by AlexReedy** [radical-collaboration/facts#249]
After todays PROTECT-FACTS meeting it's clear that we need to set up some tutorial sessions for FACTS use, specifically in the following categories:

1. FACTS Set Up
2. Using config.yml to construct experiments
3. Most probably debugging in the pilot as well (at least until we get those error logs outputted as per issue https://github.com/radical-collaboration/facts/issues/235) 
4. Discussion of RCT and it's use in FACTS

These are some basic ideas, and more can/should be added as well as setting up a time to actually do this. We should also start linking / directing folks more readily to the RCT troubleshooting resources

**Comment by shantenujha** [comment]
Thanks, @AlexReedy. Do you have a link to the PROTECT project/scope?

**Comment by Timh37** [comment]
Perhaps it would also be helpful if you could recommend some basic (installation) steps to go through prior to such a tutorial?

**Comment by AlexReedy** [comment]
@Timh37 yes good call, we do have a quick start page for FACTS that we can attach
https://fact-sealevel.readthedocs.io/en/latest/quickstart.html

**Comment by bobkopp** [comment]
And while 1 is currently addressed in the docs, and the technical specification for config.yml is also described, it would be good to add doc pages addressing 2-4 from a practical perspective.
